src/data/ts/de.ts

```diff
@@ -86,12 +86,12 @@ Beschreibung: &quot;%2&quot;</translation>
     <message>
         <location line="+8"/>
         <source>%1 has joined.</source>
-        <translation>%1 ist beigetreten</translation>
+        <translation>%1 ist beigetreten.</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>%1 has left.</source>
-        <translation>%1 ist gegangen</translation>
+        <translation>%1 ist gegangen.</translation>
```

Comment:
I've addressed this in https://github.com/nitroshare/nitroshare-desktop/pull/235